content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -200,11 +209,42 @@ spec:
 You may also be able to mutate the incoming Pod, at admission time, to unset
 the `.spec.nodeName` field and to use a node selector instead.
 
+## Admin access
+
+{{< feature-state feature_gate_name="DRAAdminAccess" >}}
+
+A ResourceClaim with admin access grants access to devices which are in use and
+may enable additional permissions when making the device available in a
+container:
+
+```yaml
+apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaimTemplate
+metadata:
+  name: large-black-cat-claim-template
+spec:
+  spec:
+    devices:
+      requests:
+      - name: req-0
+        deviceClassName: resource.example.com
+        adminAccess: true
```

Comment:
This would be easier to document if the field were named, eg, `considerAsPrivileged` or `privileged` or `limitedAccess`.

Reply:
cc @thockin @klueska 

For consideration in future APIs...